### DISCUSSION TITLE
Does applying for an Octernship automatically put you as accepted?

### DISCUSSION THREAD

**Discussion by Liftoff-Studios** [community/community#49913] [GitHub Education] [answered]
I just applied to an Octernship, and this is the message shown
>  You have successfully applied to Appwrite! Accomplish the task to apply the project. 

So does this mean that I've been accepted and the moment I complete the project I will receive payment ?

Regards

**Comment by bkpecho** [comment, marked as answer]
Hey @Liftoff-Studios 

Please note that you have only been qualified to apply and completing the task **does not mean you will receive payment.**

Completing the task provided is just part of the application process and does not guarantee acceptance into the program.

I wish you the best of luck with your application and I hope you find my answer helpful. Have a great day!

**Reply by Liftoff-Studios** [reply]
@bkpecho Thank you for answering! Does this mean the task shown on the pane of the externship page is the challenge to get accepted into the program?

Because the repo I've been invited to doesnt have any instructions for the task

**Reply by bkpecho** [reply]
It's possible that the task shown on the externship page is the challenge to getting accepted into the program. However, it's also possible that the task is simply an example of the kind of work you might be doing during the program.

If the repository you've been invited to doesn't have instructions for the task, **contact the program coordinator or the person who invited you for guidance.**

You're welcome @Liftoff-Studios. Feel free to mark my answer if it helped you 😊
